### PR TITLE
[refer dubbo 2.7.6] attachment type from map[string]stiring to map[string]interface{} 

### PR DESCRIPTION
**Pull request by cvictory**
1. attachment类型从attachment type from map[string]stiring 到 map[string]interface{} ，和dubbo java对齐
2.功能分成两部分，一部分会在dubbo-go里提交，一部分需要修改hessian.

### CODE REVIEW

**Comment by AlexStocks** [comment]
@cvictory pls check the travis failure.

**Comment by cvictory** [comment]
> @cvictory pls check the travis failure.

找了好久，没找到错误原因啊。

**Comment by wongoo** [comment]
> > @cvictory pls check the travis failure.
> 
> 找了好久，没找到错误原因啊。

```
$ go fmt && [[ -z `git status -s` ]]
binary.go
response.go
The command "go fmt && [[ -z `git status -s` ]]" exited with 1.

The command "/tmp/tools/license/license-header-checker -v -a -r -i vendor  /tmp/tools/license/license.txt . go  && [[ -z `git status -s` ]]" exited with 1.

```

File: request.go [hunk 1]
```diff
@@ -345,6 +345,8 @@ func ToMapStringString(origin map[interface{}]interface{}) map[string]string {
 
 			if vv, ok := v.(string); ok {
```
Comment (wongoo) [inline request.go:346]: not need to check now， just assign

**Comment by cvictory** [comment]
> > > @cvictory pls check the travis failure.
> > 
> > 
> > 找了好久，没找到错误原因啊。
> 
> ```
> $ go fmt && [[ -z `git status -s` ]]
> binary.go
> response.go
> The command "go fmt && [[ -z `git status -s` ]]" exited with 1.
> 
> The command "/tmp/tools/license/license-header-checker -v -a -r -i vendor  /tmp/tools/license/license.txt . go  && [[ -z `git status -s` ]]" exited with 1.
> ```

发现是有一个文件没有格式化。。

File: request.go (hunk 1 above)
Comment (cvictory) [reply request.go]: done

**Review by zouyx — APPROVED** [review]

**Pull request merged**